### ISSUE TITLE
yum update jubatus may fail when both jubatus-client and jubatus-devel is previously installed

### ISSUE THREAD

**Issue by kmaehashi** [jubatus/jubatus#1059]
Currently Jubatus client headers are contained in both `jubatus-client` and `jubatus-devel` RPM package (c91f26bffad16bae1ea3e973fba4e40b63886a84 and #271).
This causes RPM update failure in the following scenario:

1. (After Jubatus 0.8.3 release) `yum install jubatus jubatus-client jubatus-devel`
1. (After Jubatus 0.8.4 release) `yum update jubatus`

The error message is:

```
Transaction Check Error:
  file /usr/include/jubatus/client/nearest_neighbor_client.hpp from install of jubatus-devel-0.8.4-1.el6.x86_64 conflicts with file from package jubatus-client-0.8.3-1.el6.x86_64
  file /usr/include/jubatus/client/nearest_neighbor_types.hpp from install of jubatus-devel-0.8.4-1.el6.x86_64 conflicts with file from package jubatus-client-0.8.3-1.el6.x86_64
```

As `jubatus-client` package does not depend on `jubatus` package nor `jubatus-devel` package, it is not updated automatically.

We should exclude client headers from `jubatus-devel` package.

**Comment by kmaehashi** [comment]
If you face this error, you can workaround the problem by reinstalling 2 packages.

```
yum remove jubatus-client jubatus-devel
yum install jubatus-client jubatus-devel
```

This issue only affects to users installing jubatus-devel, which is only needed to build Jubatus from source or his own services.

**Comment by kmaehashi** [comment]
Fixed via #1062
